app/code/Magento/Sales/Model/Order/Shipment.php

```diff
@@ -131,7 +155,13 @@ public function __construct(
         $this->_trackCollectionFactory = $trackCollectionFactory;
         $this->_commentFactory = $commentFactory;
         $this->_commentCollectionFactory = $commentCollectionFactory;
+        $this->searchCriteriaBuilder = $searchCriteriaBuilder;
+        $this->shipmentCommentRepository = $shipmentCommentRepository;
```

Comment:
Shouldn't these two lines be removed? They are being duplicated below I think.

Reply:
Thanks, @hostep, forgot about them, would remove tomorrow.

Reply:
Thanks @hostep ,
I did not notice them from the diff :)